middleware.js

```diff
@@ -15,6 +16,17 @@ module.exports.isLoggedIn = (req, res, next) => {
   next();
 };
 
+function generateUniqueRandomArr() {
```

Comment:
This function could be moved to a seperate file, maybe helpers.js or something because I feel like it doesn't belong in here.

Reply:
Is there any common practice regarding where it would be better to keep a file like that? (should i have it in the main folder where middleware.js is, or would it be better to stash it away in /public/javascripts?

Reply:
I think in general, `middleware.js` could be kept in a folder `middleware` and `helpers.js` being there next to it.